src/main/java/Ally/Tasks/Event.java

```diff
@@ -0,0 +1,34 @@
+package Ally.Tasks;
+
+public class Event extends Task {
+    protected String from;
+    protected String to;
+
+    /**
+     * Constructor for Ally.Tasks.Event.
+     * @param description
+     * @param from
+     * @param to
+     */
+    public Event(String description, String from, String to) {
+        super(description);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String formatFile() {
+        return "E" + " | " + (isDone ? "1" : "0") + " | " + description + " | "
```

Comment:
A little tip: I find `String::format` very useful when I want to put Strings and variables together. For instance, `String.format("E | %s | %s | %s | %s", isDone ? "1" : "0", description, from, to);`.